sdcm/cluster.py

```diff
@@ -3242,7 +3242,7 @@ def node_setup(node):
         time_elapsed = time.time() - start_time
         self.log.debug('Setup duration -> %s s', int(time_elapsed))
         if not node_list[0].scylla_version:
-            result = node_list[0].remoter.run("rpm -q scylla")
+            result = node_list[0].remoter.run("rpm -q scylla-enterprise || rpm -q scylla")
```

Comment:
I agree to apply your fix first, it blocked the enterprise tests.

We can consider to use scylla_pkg() in another PR.